vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,125 @@
+# Introduction
+Entando Customer Portal is a self-service application for customers to interact within your organization to administer your Entando platform and its users. You can manage users, projects and customers. You can track projects and service tickets for your application. It is a Kubernetes-native solution built on Entando’s Application Composition Platform.
```

Comment:
"The Entando Customer Portal provides your organization with a self-service application, making it easy to manage users, projects and customers. Using the Role Based Access Control (RBAC) model, users are granted varying levels of access to optimize your workflow process. It is a Kubernetes-native solution built on Entando’s Application Composition Platform."

How's this? I don't like to start with the Tech verbiage because that seems less important than what they get from the product. Although I am not explaining it very well.

Reply:
I agree, definitely start with the business value. What you have is much better already. Tweaking further "The Entando Customer Portal enables an organization to quickly provide a modern, customer-facing self-service application for management of subscriptions and service tickets, via a lightweight integration to JIRA Service Management. Organization admins can use the included Role Based Access Control (RBAC) design to grant users different levels of access to the portal. It is a K8s..."

Reply:
definitely want to lead with value! i really like the edits

Reply:
Round 5(?)
`
The Entando Customer Portal enables an organization to quickly provide a modern, self-service customer-facing application for management of subscriptions. It includes a lightweight integration to JIRA Service Management for access to service tickets and a Role Based Access Control (RBAC) design for granting users varying levels of access.
`
1. I dropped the last sentence since it's on the Entando page and basically implied. 2. Integrations are always "to" or "with" something, rather than "on" . 3. RBAC is independent of JIRA so I separated those clauses. 4. I'm going to add professional, informative, unapproachable to my business cards. ;)

Reply:
🤣